docs/_templates/sidebar-link-items.html

```diff
@@ -17,6 +17,9 @@
                 <a class="reference internal" href="{{ 'release/release_' + version.replace('.', '_') + '.html' }}">Latest release notes</a>
                 {% endif %}
             </li>
+            <li class="toctree-11">
```

Comment:
```suggestion
            <li class="toctree-l1">
```
WOW this was subtle! 😅